pkg/sfu/streamtracker.go

```diff
@@ -103,6 +104,13 @@ func (s *StreamTracker) Observe(sn uint16) {
 	if s.paused.get() {
 		return
 	}
+
+	if !s.initialized.get() {
```

Comment:
this is probably fine.. since Observe is called from a single goroutine. but if it was called from multiple, there could be a race.